src/cli.js

```diff
@@ -268,8 +309,14 @@ export default async function run(tyr) {
     }
 
     // sign in to third party tools
-    const credentials = await signInToThirdPartyTools(configs);
-    configs.credentials = credentials;
+    const toolConfigs = await signInToThirdPartyTools(configs);
+    configs.credentials = toolConfigs.credentials;
+    if (configs.tooling.deployment === constants.heroku.name) {
+      configs.projectConfigurations.projectName = toolConfigs.appName;
```

Comment:
Can you explain why we're changing the project name here? If I understand correctly, the Heroku `appName` fails if there already exists an app with that name on Heroku, in which case it prompts them for a new app name. However, this might disagree with what they put in their configuration file when they go about setting up the project. Are there going to be issues that arise from this? Maybe we create a distinction between `projectName` and `herokuAppName` so that the project name isn't constrained by what app names are available on Heroku. Thoughts?

Reply:
Sticking with same name for local project, github repo, and heroku app name for now - for consistency